web/src/components/MapPopup.tsx

```diff
@@ -56,7 +56,7 @@ const MapPopup: React.FC<MapPopupProps> = ({ properties }: MapPopupProps): React
   return (
     <CleanLink to={path}>
       <Popup>
-        {thumbnail !== 'null' ? <PopupThumbnail src={thumbnail} /> : <Placeholder />}
+        {thumbnail !== 'null' ? <PopupThumbnail src={thumbnail} alt='' /> : <Placeholder />}
```

Comment:
What is the sense of using the empty alt?

Reply:
In generell you should set alt tags for describing pictures. 
You can leave `alt=''` if the picture has only visual purpose 
Maybe in this case we should add a todo & disabled linting because we don't get an alt tag.

Reply:
I think no alt tag is fine here since it not really has a purpose (the purpose is just adding a visual for the poi).

Reply:
yes but it's best practise to use empty alt tag

Reply:
Sorry, I meant an empty string as alt tag is fine here, you are right oc :)